File: LeetCode Questions/C++/1. Problems/1-twoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> map;
        vector<int> answer;
        for(int i=0; i<nums.size(); i++){
            int temp = target - nums[i];
            if(map.find(temp) != map.end()){
                answer.push_back(map[temp]);
                answer.push_back(i);
                break;
            }
            else{
                map[nums[i]] = i;
            }
        }
        return answer;
        
    }
};

File: LeetCode Questions/Python/1. Problems/1_two_sum.py
# https://leetcode.com/problems/two-sum/

class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        hashmap = {}
        for i in range(len(nums)):
            if target - nums[i] in hashmap:
                return hashmap[target - nums[i]], i
            else:
                hashmap[nums[i]] = i